spec.emu

```diff
@@ -416,7 +416,7 @@ contributors: Mark S. Miller, Richard Gibson
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It is used to create an immutable ArrayBuffer (i.e., an ArrayBuffer with a an [[ArrayBufferIsImmutable]] slot) with contents from _fromBlock_.</dd>
+            <dd>It is used to create an immutable ArrayBuffer (i.e., an ArrayBuffer with a an [[ArrayBufferIsImmutable]] slot) with contents from _fromBlock_. The contents of an immutable ArrayBuffer's Data Block are constrained to be permanently stable, and may not be modified by either ECMAScript code or by other activities that may take place inside an implementation or observable by it.</dd>
```

Comment:
Grammar?

```suggestion
            <dd>It is used to create an immutable ArrayBuffer (i.e., an ArrayBuffer with a an [[ArrayBufferIsImmutable]] slot) with contents from _fromBlock_. The contents of an immutable ArrayBuffer's Data Block are constrained to be permanently stable, and may not be modified by either ECMAScript code or by other activities that may take place inside an implementation or be observable by it.</dd>
```

Reply:
Is it not that the contents must never be observably modified, specifically from the perspective of ecmascript code? in practice that would still mean the host can't modify the backing memory.